components/cam/src/physics/rrtmg/radiation.F90

```diff
@@ -758,17 +756,20 @@ subroutine radiation_init(phys_state)
             call add_default('LWCF'//diag(icall),  1, ' ')
          endif
 
-         ! Add cloud-scale radiative quantities
-         if (use_SPCAM) then
-            call addfld ('CRM_QRAD', (/'crm_nx_rad','crm_ny_rad','crm_nz    '/), 'A', 'K/s', 'Radiative heating tendency')
-            call addfld ('CRM_QRS ', (/'crm_nx_rad','crm_ny_rad','crm_nz    '/), 'I', 'K/s', 'CRM Shortwave radiative heating rate')
-            call addfld ('CRM_QRSC', (/'crm_nx_rad','crm_ny_rad','crm_nz    '/), 'I', 'K/s', 'CRM Clearsky shortwave radiative heating rate')
-            call addfld ('CRM_QRL ', (/'crm_nx_rad','crm_ny_rad','crm_nz    '/), 'I', 'K/s', 'CRM Longwave radiative heating rate' )
-            call addfld ('CRM_QRLC', (/'crm_nx_rad','crm_ny_rad','crm_nz    '/), 'I', 'K/s', 'CRM Longwave radiative heating rate' )
-            call addfld ('CRM_CLD_RAD', (/'crm_nx_rad','crm_ny_rad','crm_nz    '/), 'I', 'fraction', 'CRM cloud fraction' )
-         end if
-       end if
-    end do
+       end if  ! active_calls(icall)
+    end do  ! icall = 0,N_DIAG
+
+    ! Add cloud-scale radiative quantities
```

Comment:
These were previously inside the `if (active_calls(icall))` check for the rad "diagnostics" calls, but shouldn't have been.

Reply:
ok, I get it.